aten/src/THC/generic/THCTensor.cpp

```diff
@@ -446,8 +444,8 @@ void THCTensor_(unfold)(THCState *state, THCTensor *self, THCTensor *src, int di
 
   THCTensor_(set)(state, self, src);
 
-  newSize = (int64_t*)THAlloc(sizeof(int64_t)*(self->dim()+1));
-  newStride = (int64_t*)THAlloc(sizeof(int64_t)*(self->dim()+1));
+  std::vector<int64_t> newSize(/* size */ self->dim() + 1);
```

Comment:
I know we have this sort of thing in other places, but I just don't get it.  Any reasonable editor can tell you what this parameter is, and you break commenting out blocks of code to get it.  But maybe I'm missing something.

Reply:
I'm just twitchy about the difference between `std::vector<int64_t> a(1)` and `std::vector<int64_t> a{1}`. I can delete the comments. (`#if 0 ... #endif` for block commenting, though? ;)

Reply:
ok you caught me with my own argument, because editors are usually too dumb to display this like a comment :).